src/lib.rs

```diff
@@ -136,7 +153,7 @@ impl<'a, K, V> FixieTrie<K, V> where K: FixedLengthKey {
 
     fn new_tuple_twig(key: K, value: V) -> TriePtr {
         unsafe {
-            let p = heap::allocate(mem::size_of::<(K,V)>(),
+            let p = allocate(mem::size_of::<(K,V)>(),
                                    mem::align_of::<(K,V)>()) as *mut (K,V);
```

Comment:
Although it's more churn, I'd prefer that the indentation be fixed here and everywhere else that it changes.

Reply:
Sure. 
Which reminds me, I should look into a `rustfmt` git pre-commit hook.